cves/kernel/CVE-2016-6130.yml

```diff
@@ -356,9 +352,9 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
-order_of_operations:
+  answer: false
+  note: Not applicable
```

Comment:
Elaborate on your note. Why it's not applicable? Talk about how you came to the conclusions you did

Reply:
All done! Thanks for pointing it out.